.github/auto-merge.yml

```diff
@@ -0,0 +1,38 @@
+# Configuration for probot-auto-merge - https://github.com/bobvanderlinden/probot-auto-merge
+
+# Whether an out-of-date pull request is automatically updated.
+# It does so by merging its base on top of the head of the pull request.
+# This is the equivalent of clicking the 'Update branch' button.
+# This is useful for repositories where protected branches are used and the option
+# 'Require branches to be up to date before merging' is enabled.
+# Note: this only works when the branch of the pull request resides in the same repository as
+#       the pull request itself.
+updateBranch: true
+
+# Whether the pull request branch is automatically deleted.
+# This is the equivalent of clicking the 'Delete branch' button shown on merged pull requests.
+# Note: this only works when the branch of the pull request resides in the same repository as
+#       the pull request itself.
+deleteBranchAfterMerge: true
+
+# In what way a pull request needs to be merged. This can be:
+# * merge: creates a merge commit, combining the commits from the pull request on top of
+#   the base of the pull request (default)
+# * rebase: places the commits from the pull request individually on top of the base of the pull request
+# * squash: combines all changes from the pull request into a single commit and places the commit on top
+#   of the base of the pull request
+# For more information see https://help.github.com/articles/about-pull-request-merges/
+mergeMethod: squash
+
+# Blocking labels are the labels that can be attached to a pull request to make sure the pull request
+# is not being automatically merged.
+blockingLabels:
+- on hold
+
+# Whenever required labels are configured, pull requests will only be automatically merged whenever
+# all of these labels are attached to a pull request.
+requiredLabels:
+- automerge
+
+# Automatic merges will be blocked if there is a match between the regular expression and title
+blockingTitleRegex: '\bWIP\b'
```

Comment:
Do we want to make this case insensitive?

Reply:
[It's case insensitive](https://github.com/bobvanderlinden/probot-auto-merge#blockingtitleregex-condition-default-none). I think that's fine.

Reply:
Well not in PCRE, but I don't know what they're doing under the covers. It doesn't matter I agree this is fine then.